src/js/utils/captions.js

```diff
@@ -271,16 +272,38 @@ export const toggleSubsCaps = (el, force) => {
       el.dispatchEvent(evt);
     }
   } else if (!subsOn || force === true) {
-    // Subtitles are off. Clicking should show the first relevant captions track or subtitles track (true/"on" by default)
-    const [subTrackStr] =
-      splitTextTracksStr(
+    let subTrackObjs = el?.mediaSubtitlesList ?? [];
+    if (!subTrackObjs?.length && el.hasAttribute(MediaUIAttributes.MEDIA_SUBTITLES_LIST)) {
+      subTrackObjs = parseTextTracksStr(
         el.getAttribute(MediaUIAttributes.MEDIA_SUBTITLES_LIST) ?? ''
-      ) ?? [];
-    if (subTrackStr) {
+      );
+    }
+
+    let subTrack = subTrackObjs[0];
+    if (!noSubtitlesPref) {
+      const subtitlesPref = globalThis.localStorage.getItem('media-chrome-pref-subtitles');
```

Comment:
Since this is for languages let's make the pref name more specific, like `media-chrome-pref-subtitles-lang`. I can imagine other subtitle related prefs.

Should the pref be across all tracks? `media-chrome-pref-texttrack-lang`

Reply:
Are you thinking `texttrack` because you envision going forward that this would also apply to e.g. `chapters` or `metadata` text tracks with a defined language? Based on our prior convos around "app"/"player" language (e.g. dialogs, tooltips, A11Y language, etc.), I can see arguments for treating these cases more like those cases. On the other hand, these would likely be "media specific (aka strongly tied to a particular media `src`), which roughly resembles subtitles/cc. My S1 is to treat it like our (yet to be implemented) "app"/"player" language settings (likely tied to the `lang` attribute/prop), but I'm down to discuss since it's def less clear cut than the others. If we stick with my S1, I'd lean towards `media-chrome-pref-subtitles-lang` (or similar).

Or, was that a slip and you didn't actually intend to swap out `subtitles` for `texttrack`?

Reply:
Not a slip, just exploring if once a user sets a lang pref in subtitles if it's be a positive user experience to default other tracks to that lang when possible. I think there's something there, but no need to solve here. media-chrome-pref-subtitles-lang 👍